apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -33,6 +34,21 @@ export function Workspace({
         ${HeadContents({ resLocals, pageNote, pageTitle })}
         <link href="${assetPath('stylesheets/workspace.css')}" rel="stylesheet" />
         ${compiledScriptTag('workspaceClient.ts')}
+        ${resLocals.assessment &&
+        resLocals.assessment.type === 'Exam' &&
```

Comment:
Nit, slightly more concise:

```suggestion
        ${resLocals.assessment?.type === 'Exam' &&
```

Reply:
Changed.